doc/api/crypto.md

```diff
@@ -2875,6 +2875,18 @@ added: v15.6.0
 
 The date/time until which this certificate is valid.
 
+### `x509.validFromDate`
+
```

Comment:
When adding an API, include the appropriate YAML documentation with REPLACEME for the version number.

For example:
```
### `request.method`

<!-- YAML
added: REPLACEME
-->

* {string} The request method.
```